.env.example

```
@@ -37,6 +37,10 @@ OAUTH_GOOGLE_CLIENT_ID=xxx-xxx.apps.googleusercontent.com
 OAUTH_GOOGLE_CLIENT_SECRET=GOCSPX-xxx
 OAUTH_GOOGLE_REDIRECT_URL=http://localhost:5173/auth/google/callback
 
+PUSH_VAPID_PUBLIC_KEY=your-vapid-public-key
+PUSH_VAPID_PRIVATE_KEY=your-vapid-private-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PUSH_VAPID_PRIVATE_KEY key should go before the PUSH_VAPID_PUBLIC_KEY key

(UnorderedKey)